cfme/storage/volume.py

```diff
@@ -243,6 +287,31 @@ def create_snapshot(self, name, cancel=False, reset=False):
                 view.save.click()
                 return snapshot_collection.instantiate(name, self.provider)
 
+    def attach_instance(self, name, mountpoint=None, cancel=False, reset=False):
+        view = navigate_to(self, 'AttachInstance')
+        changed = view.fill({'instance': name, 'mountpoint': mountpoint})
+
+        if changed:
+            if reset:
+                view.reset.click()
+            elif cancel:
```

Comment:
**Required** The cancel button is always active and not dependent on the view changing. 

Also, I would add a comment saying that reset and attach are only active when the view is changed